simtools/simtel/simtel_io_histogram.py

```diff
@@ -50,26 +50,35 @@ class SimtelIOHistogram:
         calculating trigger rate for individual telescopes.
         If false, the area thrown is estimated based on the maximum distance as given in
         the simulation configuration.
-
+    energy_range: list
+        The energy range used in the simulation. It must be passed as a list of floats and the
+        energy must be in TeV (as in the CORSIKA configuration).
```

Comment:
Important: CORSIKA usually works in GeV. Are you sure about the TeV here?

Reply:
Good point. The docstring is confusing, hence I removed the parentheses. But the code is correct. E_range is already given in TeV (this is dealt with in pyeventio).